src/main/java/com/kodilla/ecommercee/domain/CartDto.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CartDto {
+    private Long id;
```

Comment:
Pola nie są zgodne z jsonem https://github.com/pplutap/JDP-2003-02/blob/master/specs/cart.spec.json

Reply:
Got it ;D

Reply:
fixed